recipe/meta.yaml

```diff
@@ -12,33 +12,38 @@ source:
         # Skips 3 tests (check_extension, check_sql_stmt, and check_get_normal_zoom_extension_load) that are failing on OS X.
         # Re-visit this once a new release is out.
         - skip_failing_test.patch  # [osx]
+        - makefile.vc.patch  # [win]
+        - config-msvc.patch  # [win]
+        - gg_extras.patch  # [win]
 
 build:
-    number: 7
-    skip: True  # [win]
+    number: 8
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
 
 requirements:
     build:
-        - sqlite 3.9.*
+        - sqlite
```

Comment:
Let's pin to the working version instead of leaving it unpinned. Also, if this works, we need to update the pinnings in: https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L45